app/tests/unit/components/Form/ProjectForm.test.tsx

```diff
@@ -23,46 +20,108 @@ const loadedQuery = graphql`
   }
 `;
 
-let environment;
-const TestRenderer = (extraProps: any) => {
-  const data = useLazyLoadQuery<ProjectFormQuery>(loadedQuery, {});
-  return (
-    <ProjectForm
-      query={data.query}
-      projectRevision={data.query.projectRevision}
-      setValidatingForm={jest.fn()}
-      {...extraProps}
-    />
-  );
+const mockQueryPayload = {
+  ProjectRevision() {
+    const result: ProjectForm_projectRevision = {
+      " $fragmentType": "ProjectForm_projectRevision",
```

Comment:
```suggestion
    const result: Partial<ProjectForm_projectRevision> = {
```

This applies to the other resolvers below